x/evm/keeper/keeper.go

```diff
@@ -97,10 +97,6 @@ func (k *Keeper) GetBlockBloomMapping(ctx sdk.Context, height int64) (ethtypes.B
 	}
 
 	bloom := store.Get(types.BloomKey(bz))
-	if len(bloom) == 0 {
-		return ethtypes.BytesToBloom([]byte{}), fmt.Errorf("block with bloombits %v not found", bloom)
-	}
-
```

Comment:
undo change

Reply:
this causes the getFilterChanges function to fail since it causes getEthBlockByNumber to fail, I suspect the bloom mapping isn't getting set properly. should i fix it in this PR?